### ISSUE TITLE
Talkback not working on Assignment page

### ISSUE THREAD

**Issue by poojaustad** [UstadMobile/UstadMobile#347]
**Describe the bug**
On assignments page the talkback is not working as expected

**To Reproduce**
1. Talkback is ON
2. Teacher user on Course edit tries to create a new assignment
3. When the user selects the assignment option- the talkback stops responding 

**Expected behavior**
On the Assignment page, the user should be able to access talkback

**Screenrecord**

https://user-images.githubusercontent.com/78338755/176155035-29ee2d5f-51c8-4b66-b04e-b998ecb0f147.mp4

Noted on Android device
